ci/php-8_1/devbox.json

```diff
@@ -0,0 +1,15 @@
+{
+  "packages": [
+    "php@8.1",
+    "php81Packages.composer@2.5.5"
+  ],
+  "shell": {
+    "init_hook": [
+      "cd ../..",
```

Comment:
`devbox run` will put us into the directory where `devbox.json` is, so we need to put ourselves back in the root directory.